assets/javascripts/kitten/components/navigation/nav-bar/index.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import classNames from 'classnames'
+import COLORS from '../../../constants/colors-config'
+import TYPOGRAPHY from '../../../constants/typography-config'
+import { pxToRem, stepToRem } from '../../../helpers/utils/typography'
+import {
+  CONTAINER_PADDING_THIN,
+  CONTAINER_PADDING,
+} from '../../../constants/grid-config'
+import { ScreenConfig } from '../../../constants/screen-config'
+
+const StyledNavBar = styled.div`
+  width: auto;
+  flex-grow: 1;
+
+  .k-NavBar__nav {
+    width: auto;
+    margin: 0;
+    padding: 0 ${pxToRem(CONTAINER_PADDING_THIN)};
+    background: ${({ colors }) => colors.background || COLORS.primary6};
+    display: flex;
+    justify-content: center;
+
+    @media (min-width: ${pxToRem(ScreenConfig.S.min)}){
+      padding: 0 ${pxToRem(CONTAINER_PADDING)};
+    }
+
+    &::before, &::after {
+      content: "";
+      flex-grow: 1;
+    }
+  }
+
+  .k-NavBar__list {
+    list-style-type: none;
+    display: flex;
+    align-items: center;
+
+    margin: 0;
+    padding: 0;
+  }
+
+  .k-NavBar__listItem + .k-NavBar__listItem {
+    margin-left: ${pxToRem(25)};
+
+    @media (min-width: ${pxToRem(ScreenConfig.S.min)}){
+      margin-left: ${pxToRem(50)};
+    }
+  }
+
+  .k-NavBar__link {
+    height: ${pxToRem(65)};
+    box-sizing: border-box;
+    ${TYPOGRAPHY.fontStyles.regular}
+    font-size: ${stepToRem(-1)};
+    text-decoration: none;
+    text-align: center;
+
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: row;
+    position: relative;
+    outline: none;
+    color: ${({ colors }) => colors.link || COLORS.font1};
+    border-bottom: ${pxToRem(4)} solid ${({ colors }) =>
+  colors.border || 'transparent'};
```

Comment:
L'indentation est bizarre ici.
```suggestion
      colors.border || 'transparent'};
```